AMQP/rabbitmq/gateway/Dockerfile

```diff
@@ -23,15 +22,10 @@ RUN set -ex \
 
 ENV NODE_VERSION 0.10.45
 
-RUN curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-x64.tar.xz" \
-  && curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/SHASUMS256.txt.asc" \
-  && gpg --batch --decrypt --output SHASUMS256.txt SHASUMS256.txt.asc \
-  && grep " node-v$NODE_VERSION-linux-x64.tar.xz\$" SHASUMS256.txt | sha256sum -c - \
-  && tar -xJf "node-v$NODE_VERSION-linux-x64.tar.xz" -C /usr/local --strip-components=1 \
-  && rm "node-v$NODE_VERSION-linux-x64.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt
-
-RUN git clone https://github.com/kaazing/javascript.client.tutorials ./web/javascript.client.tutorials
-RUN npm --prefix ./web/javascript.client.tutorials/amqp install ./web/javascript.client.tutorials/amqp 
+RUN curl -LOk  https://github.com/kaazing/javascript.client.tutorials/files/324464/websocket-jms-tutorial.zip
```

Comment:
@dpwspoon I've changed the Dockerfiles such that there is a single RUN command for downloading and unzipping the tutorials to the right locations.
Regarding the check to be secure, I assume you referred to the -k flag that I added, which doesn't check for the certificate, but if I don't use it, the tutorial is not downloaded.